boot/zephyr/Kconfig

```diff
@@ -320,6 +307,31 @@ config BOOT_SWAP_SAVE_ENCTLV
 
 endif # !SINGLE_APPLICATION_SLOT
 
+if SINGLE_APPLICATION_SLOT
+
+config SINGLE_APPLICATION_SLOT_RAM_LOAD
+	bool "RAM load for single application slot"
+	help
+	  If y, the image is loaded to RAM and executed from there. For this reason,
+	  the image has to be linked to be executed from RAM. The address that the
+	  image is copied to is specified using the load-addr argument to the
+	  imgtool.py script which writes it to the image header.
+
+endif # SINGLE_APPLICATION_SLOT
```

Comment:
this needs to be a distinct mode, not one that uses `SINGLE_APPLICATION_SLOT`, that Kconfig is as I said in the previous review for single flash application only, there needs to be a new Kconfig, one that does not use an existing Kconfig

Reply:
Ok - moved the Kconfig outside `if SINGLE_APPLICATION_SLOT`, so one doesn't need something like: 

```
CONFIG_SINGLE_APPLICATION_SLOT=y
CONFIG_SINGLE_APPLICATION_SLOT_RAM_LOAD=y
```

Only `CONFIG_SINGLE_APPLICATION_SLOT_RAM_LOAD=y` is needed. Implementation-wise though, it will select `SINGLE_APPLICATION_SLOT` as the code path is basically the same.